packages/store/test/StoreCore.t.sol

```diff
@@ -800,17 +806,6 @@ contract StoreCoreTest is Test, StoreMock {
     }
     data.newSecondDataBytes = abi.encodePacked(data.secondDataBytes, data.secondDataToPush);
 
-    // Expect a Store_SpliceDynamicData event to be emitted
-    vm.expectEmit(true, true, true, true);
-    emit Store_SpliceDynamicData(
-      data.tableId,
-      data.keyTuple,
-      uint48(data.secondDataBytes.length),
-      0,
-      PackedCounterLib.pack(data.newSecondDataBytes.length, data.thirdDataBytes.length),
-      data.secondDataToPush
-    );
-
```

Comment:
❗ temporarily deleted these tests until I figure out what `startWithinField` should be for push/pop

Reply:
Added them back